code/responsive-design/style.css

```diff
@@ -0,0 +1,106 @@
+body {
+  margin: 0;
+}
+
+#profile {
+  background-color: #f9b8b6;
+  padding: 1em;
+  margin: 1em;
+}
+
+#profile > img {
+  outline: 0.3rem solid #fff;
+  margin: 0.5em 0;
+}
+
+#profile > span {
+  font-size: 1.3em;
+  font-weight: 600;
+  padding-bottom: 1em;
+}
+
+.card {
+  border: 3px solid #006d9e;
+  margin: 0.2rem;
+}
+
+img {
+  width: 100%;
+}
+
+header {
+  background-color: #a7cddf;
+  font-size: 2em;
+  display: flex;
+  justify-content: center;
+}
+/* media queries */
+@media screen and (min-width: 500px) {
+  #gallery {
+    /* flex properties */
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  .card {
+    /* style properties */
+    width: 90%;
+    /* flex properties */
+    display: flex;
+  }
+  .card img {
+    width: 70%;
+  }
+  .card .photo-label {
+    padding: 1rem;
+  }
+  .right img {
+    order: 2;
+  }
+}
+@media screen and (min-width: 750px), (min-width: 1000px) {
```

Comment:
It's not necessary to put multiple min-widths, only the smaller one would be applied. If you want to indicate a range you could combine a min and a max, but it's better to let the cascade handle ranges. In the future just put one size indication!